ios/ci_scripts/ci_pre_xcodebuild.sh

```
@@ -0,0 +1,79 @@
+
+# Configure Sentry CLI
+export SENTRY_AUTH_TOKEN=$SENTRY_AUTH_TOKEN
+export SENTRY_ORG=$SENTRY_ORG
+export SENTRY_PROJECT=$SENTRY_PROJECT
+
+# Install Sentry CLI without Homebrew auto-update
+HOMEBREW_NO_AUTO_UPDATE=1 brew install getsentry/tools/sentry-cli
+
+# Debug information
+echo "Current directory: $(pwd)"
+echo "Sentry Auth Token: ${SENTRY_AUTH_TOKEN:0:8}..."
```

Shellcheck

[warning] In POSIX sh, string indexing is undefined.

(SC3057)